Sources/do-not-disturb/DoNotDisturb.swift

```diff
@@ -12,22 +12,35 @@ public struct DoNotDisturb {
 		DistributedNotificationCenter.default().postNotificationName(NSNotification.Name("com.apple.notificationcenterui.dndprefs_changed"), object: nil, deliverImmediately: true)
 	}
 
-	private static func enable() {
-		set("dndStart", value: 0 as CFPropertyList)
-		set("dndEnd", value: 1440 as CFPropertyList)
-		set("doNotDisturb", value: true as CFPropertyList)
-		commitChanges()
+	private static func restartNotificationCenter() {
+		NSRunningApplication.runningApplications(withBundleIdentifier: "com.apple.notificationcenterui").first?.forceTerminate()
+	}
 
-		// For some reason `doNotDisturb` does not take the first time around
-		// TODO: Figure out why
-		sleep(for: 0.4)
-		set("doNotDisturb", value: true as CFPropertyList)
+	private static func enable() {
+		if !isEnabled {
```

Comment:
Can you use a `guard` statement here instead (early-return)?

Reply:
f84a569